x-pack/test/functional/apps/discover/group1/reporting.ts

```diff
@@ -137,16 +141,16 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       });
 
       it('is available if new', async () => {
-        await reporting.openExportTab();
+        await reporting.openExportPopover();
```

Comment:
Please note that `x-pack/test/functional/apps/discover/group1/reporting` tests are skipped at the moment on main https://github.com/elastic/kibana/issues/219807

We should verify that they are still working as expected after these changes.
Can we try to unskip them in this PR?

Reply:
Created a separate PR to unskip tests https://github.com/elastic/kibana/pull/220826

Reply:
Thanks for the heads up @jughosta, I saw your PR to fix it. We can merge in your fix into main and I rebase.